### ISSUE TITLE
having trouble doing select on stream

### ISSUE THREAD

**Issue by Exubient** [confluentinc/ksql#1683]
I'm having trouble running a select query on a stream that I have create
The details are the following:

```
print 'topic5' from beginning:
{"ROWTIME":1533170792492,"ROWKEY":"key","gold":2042}
{"ROWTIME":1533170793493,"ROWKEY":"key","gold":2043}
{"ROWTIME":1533170794494,"ROWKEY":"key","gold":2044}
{"ROWTIME":1533170795496,"ROWKEY":"key","gold":2045}
...
```

I've created the stream this way:
`create stream goldstream(gold int) with (kafka_topic='topic5', value_format='json');`

```
describe goldstream;
 Field   | Type
-------------------------------------
 ROWTIME | BIGINT           (system)
 ROWKEY  | VARCHAR(STRING)  (system)
 GOLD    | INTEGER
-------------------------------------
```

```
list streams;
 Stream Name | Kafka Topic | Format
------------------------------------
 GOLDSTREAM       | topic5      | JSON
------------------------------------
```

the problem is that 
`SELECT * FROM goldstream;`
hangs and does not show any results

**Comment by apurvam** [comment]
Can you share the contents of your `ksql.log` file?

**Comment by Exubient** [comment]
do you mean something like this? this is the log that popped on my ksql-server
```
[2018-08-01 18:02:57,686] INFO StreamsConfig values:
        application.id = _confluent-ksql-default_transient_4007573019079144999_1533171777654
        application.server =
        bootstrap.servers = [<bootstap-ip>:9092]
        buffered.records.per.partition = 1000
        cache.max.bytes.buffering = 10485760
        client.id = _confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-92-consumer
        commit.interval.ms = 30000
        connections.max.idle.ms = 540000
        default.deserialization.exception.handler = class org.apache.kafka.streams.errors.LogAndFailExceptionHandler
        default.key.serde = class org.apache.kafka.common.serialization.Serdes$ByteArraySerde
        default.production.exception.handler = class org.apache.kafka.streams.errors.DefaultProductionExceptionHandler
        default.timestamp.extractor = class org.apache.kafka.streams.processor.FailOnInvalidTimestamp
        default.value.serde = class org.apache.kafka.common.serialization.Serdes$ByteArraySerde
        key.serde = null
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        num.standby.replicas = 0
        num.stream.threads = 1
        partition.grouper = class org.apache.kafka.streams.processor.DefaultPartitionGrouper
        poll.ms = 100
        processing.guarantee = at_least_once
        receive.buffer.bytes = 32768
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        replication.factor = 1
        request.timeout.ms = 40000
        retries = 0
        retry.backoff.ms = 100
        rocksdb.config.setter = null
        security.protocol = PLAINTEXT
        send.buffer.bytes = 131072
        state.cleanup.delay.ms = 600000
        state.dir = /tmp/kafka-streams
        timestamp.extractor = null
        upgrade.from = null
        value.serde = null
        windowstore.changelog.additional.retention.ms = 86400000
        zookeeper.connect =
 (org.apache.kafka.streams.StreamsConfig:279)
[2018-08-01 18:02:57,688] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-89] Starting (org.apache.kafka.streams.processor.internals.StreamThread:712)
[2018-08-01 18:02:57,689] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-89] State transition from CREATED to RUNNING (org.apache.kafka.streams.processor.internals.StreamThread:200)
[2018-08-01 18:02:57,688] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-90] Starting (org.apache.kafka.streams.processor.internals.StreamThread:712)
[2018-08-01 18:02:57,691] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-90] State transition from CREATED to RUNNING (org.apache.kafka.streams.processor.internals.StreamThread:200)
[2018-08-01 18:02:57,691] INFO stream-client [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea] Started Streams client (org.apache.kafka.streams.KafkaStreams:782)
[2018-08-01 18:02:57,693] INFO Streaming query 'select * from take2;' (io.confluent.ksql.rest.server.resources.streaming.StreamedQueryResource:73)
[2018-08-01 18:02:57,691] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-92] Starting (org.apache.kafka.streams.processor.internals.StreamThread:712)
[2018-08-01 18:02:57,693] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-92] State transition from CREATED to RUNNING (org.apache.kafka.streams.processor.internals.StreamThread:200)
[2018-08-01 18:02:57,692] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-91] Starting (org.apache.kafka.streams.processor.internals.StreamThread:712)
[2018-08-01 18:02:57,700] INFO stream-thread [_confluent-ksql-default_transient_4007573019079144999_1533171777654-7cfcb78b-9964-4d05-82c6-c6e5995b95ea-StreamThread-91] State transition from CREATED to RUNNING (org.apache.kafka.streams.processor.internals.StreamThread:200)

```

**Comment by apurvam** [comment]
That looks like the right log, but the output suggests you are actually running `select * from take2`, and not from `goldstream`.

**Comment by Exubient** [comment]
oh that right,  I've changed the names but the goldstream should be take2
(the names are  not the problem right now/ just think of them to be the right naming)

I was running Ksql(installed from Confluent) on a CDH multi-node environment and I'm guessing some of the resources got mixed somehow? From using Kafka from CDH and Ksql from confluent? can this be possible?

I made a brand new instance with only the confluent installation and the hanging problem is gone.
